spec/system/project_details_spec.rb

```diff
@@ -70,6 +73,12 @@
         expect(page).to have_content("RDSS")
         expect(page).to have_content("Research")
       end
+
+      it "displays a standard indicator for empty fields of an approved project" do
+        sign_in sponsor_user
+        visit "/projects/#{project_in_mediaflux.id}/details"
+        expect(page.find("#project-purpose-value").text).to eq "—"
```

Comment:
Let's see if we can change the `page.find` for `expect page to have`, see Carolyn suggestion on 2025/10/24 (https://docs.google.com/document/d/1syXKVMx-uyA3-zlMB4N9kdrcLRg-qtCPv1JeCSIduv4/edit?tab=t.0)